### DISCUSSION TITLE
How are we going to deal with PageSpeed Insight scores?

### DISCUSSION THREAD

**Discussion by stressed-pet-owner** [gatsbyjs/gatsby#35836] [Help]
So I was hired almost 2 years ago to build headless CMS sites using WordPress and Gatsby in a very very good and friendly digital agency that is developing mostly eye-candy sites.

I can easily score 90 in mobile, but this is like a very simple site with no carousel or video above the fold.

Now the WordPress team in our company developed a technique (IDK what), but they're just nailing it almost a perfect score in mobile with too much going on above the fold and the home page itself. Carousel? No worries! Video above the fold? Sure! They're killing it.

Meanwhile in Gatsby, you just add Recaptcha or GA or GTM and its mobile score plummets. I'm not even adding animations anymore.

Please note that we're doing marketing sites and they need to be "good-looking" and must have these scores in PSI.

I can't push building sites using headless CMS anymore in our company with what's going on, but I will still be rooting for Gatsby.

**Comment by KyleAMathews** [comment]
Hi, check out the new <Script> component & particularly the experimental Partytown integration — these can be really helpful in improving scores https://github.com/gatsbyjs/gatsby/discussions/35404

**Reply by stressed-pet-owner** [reply]
I'm confused though. What if I have a <ReactDatePicker> component. How can I implement it with the <Script> component?
